cmake/modules/FindAtomic.cmake

```diff
@@ -75,24 +59,17 @@ endif()
 add_library(eProsima_atomic INTERFACE)
 
 # Populate the interface target properties
-if (NOT ATOMIC_WITHOUT_LIB)
-    if (ATOMIC_WITH_LIB)
-        # force to link to atomic when the dummy target is present
-        if (NOT (CMAKE_VERSION VERSION_LESS "3.11.4"))
-            target_link_libraries(eProsima_atomic INTERFACE atomic)
-        else()
-            set_property(TARGET eProsima_atomic PROPERTY INTERFACE_LINK_LIBRARIES atomic)
-        endif()
+if (ATOMIC_WITH_LIB)
+    # force to link to atomic when the dummy target is present
+    if (NOT (CMAKE_VERSION VERSION_LESS "3.11.4"))
```

Comment:
This will always evaluate true since we raised CMake's minimum required version to 3.16.3

Reply:
It's a module. If we use it in any other project like Integration Service or Discovery Server (ancillary tool) it will fail.
But ... I've been willing to do it for so long that I will remove the thing.